messenger.rst

```diff
@@ -1694,7 +1694,7 @@ redis_sentinel           support is disabled
 ssl                      Map of TLS options.                    null
 =======================  =====================================  =================================
 
-The ``ssl`` option can be used to change requirements for the TLS channel, e.g. in tests:
+The ``ssl`` option can be used to provide SSL context options (`php.net/context.ssl`_) for the TLS channel, e.g. in tests:
```

Comment:
```suggestion
The ``ssl`` option can be used to provide `SSL context options`_ for the TLS channel, e.g. in tests:
```
WDYT about this for better link integration in text ?

Reply:
I think we could use the proposed variant if this keeps the reference intact to the PHP docs.

![image](https://github.com/user-attachments/assets/914d6466-696f-4a95-a2d2-5ad82e1fe358)

See: https://github.com/symfony/symfony-docs/blob/f39f6576b458adf4ab04726066f921557a7041cb/components/cache/adapters/redis_adapter.rst?plain=1#L220